companies/danskebank.json

```diff
@@ -0,0 +1,19 @@
+{
+    "slug": "danskebank",
+    "relevant-countries": [
+        "dk"
```

Comment:
From a quick Google search, the data in this record also applies to at least the UK and Germany, possible other countries.

Reply:
[Danske-Wiki](https://en.m.wikipedia.org/wiki/Danske_Bank) in this link i found the list of countries in which db is having its branches. Should i include all of them??
Thanks for help!!

Reply:
That seems like a good idea, yes.